lean4-infoview/src/infoview/info.tsx

```diff
@@ -319,35 +347,46 @@ function InfoAux(props: InfoProps) {
                 if (g) return updateTermGoal(g)
                 else return undefined
             }).catch(() => undefined) // ignore error on Lean version that don't support term goals yet
-            allReq = Promise.all([goalsReq, termGoalReq]);
-        }
-
-        function onError(err: any) {
-            const errS = typeof err === 'string' ? err : JSON.stringify(err);
-            // we need to check if this value is empty or not, because maybe we are assigning
-            // a message error with an empty error
-            if (errS === '{}' || errS === undefined) {
-                setError(undefined);
-            }
-            else {
-                setError(`Error fetching goals: ${errS}`);
-                setStatus('error');
-            }
+            allReq = Promise.all([
+                goalsReq,
+                termGoalReq,
+                new Promise<undefined>(resolve => resolve(undefined))
```

Comment:
```suggestion
                undefined
```
In javascript, `T` is a subtype of `Promise<T>`.  For a particularly enlightening example, check this out:
```javascript
(async () => await console.log(await await await 42))()
```